qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -50,49 +51,100 @@ def run_cephfs_shell_cmd(self, cmd, mount_x=None, shell_conf_path=None,
         args.extend(("--", cmd))
 
         log.info("Running command: {}".format(" ".join(args)))
-        return mount_x.client_remote.run(args=args, stdout=BytesIO(),
-                                         stderr=BytesIO(), stdin=stdin)
+        return mount_x.client_remote.run(args=args, stdout=StringIO(),
+                                         stderr=StringIO(), stdin=stdin,
+                                         check_status=check_status)
 
-    def get_cephfs_shell_cmd_error(self, cmd, mount_x=None,
-                                   shell_conf_path=None, opts=None,
-                                   stdin=None):
-        return ensure_str(self.run_cephfs_shell_cmd(
-            cmd=cmd, mount_x=mount_x, shell_conf_path=shell_conf_path,
-            opts=opts, stdin=stdin).stderr.getvalue().strip())
+    def negtest_cephfs_shell_cmd(self, **kwargs):
+        """
+        This method verifies that cephfs shell comamnd failed for with
+        expected return value and/or error message.
+
+        kwargs is expected to hold the arguments same as run_cephfs_shell_cmd
+        with following exceptions -
+            * should not contain check_status; it doesn'
+            * may nor may not contain error message and return value in
+              dict members 'errmsg' and 'retval' respectively.
```

Comment:
```suggestion
        kwargs is expected to hold the arguments same as run_cephfs_shell_cmd
        with following exceptions -
            * It should not contain check_status.
            * It may contain error message and return value in
              dict members 'errmsg' and 'retval' respectively.
```

Reply:
Fixed.

Reply:
Thanks!